.env.test

```diff
@@ -3,3 +3,4 @@ KERNEL_CLASS='App\Kernel'
 APP_SECRET='$ecretf0rt3st'
 SYMFONY_DEPRECATIONS_HELPER=999999
 PANTHER_APP_ENV=panther
+DATABASE_URL=postgres://postgres@127.0.0.1:3306/postgres
```

Comment:
3306 is the default port on mysql. 5432 is the one for postgres

Reply:
Yes, that should be changed, but we are not creating a DB this time around, hopefully someday soon we can add fixtures and do proper regressions testing against the UI also.

Reply:
ok, good point...